core/src/main/scala/akka/kafka/internal/KafkaConsumerActor.scala

```diff
@@ -124,7 +124,7 @@ class KafkaConsumerActor[K, V](settings: ConsumerSettings[K, V]) extends Actor w
   val pollMsg = Poll(this, periodic = true)
   val delayedPollMsg = Poll(this, periodic = false)
   def pollTimeout() = settings.pollTimeout
-  def pollInterval() = settings.pollInterval
+  def pollInterval() = 10.minutes
```

Comment:
This test was meant to illustrate the misbehaviour, not change the logic. This should be set via settings in the test.